src/DomBuilder/CustomerCreditTransferDomBuilder.php

```diff
@@ -250,6 +250,26 @@ protected function appendAddressToDomElement(\DOMElement $creditor, CustomerCred
 
         $postalAddress = $this->createElement('PstlAdr');
 
+        //Generate Street Name address node.
```

Comment:
We are not accepting any new fields without test coverage for them so please add the appropriate test cases.